packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -329,11 +329,17 @@ class _ExpansionPanelListState extends State<ExpansionPanelList> {
           widget.expansionCallback!(childIndex, false);
         }
       }
+        //Because when _handlePressed the state is not yet expanded
+      widget.expansionCallback?.call(index, !isExpanded);
 
-      setState(() {
+       setState(() {
         _currentOpenPanel = isExpanded ? null : pressedChild;
       });
     }
+    else
```

Comment:
I did not completely understand the else behavior. Can you explain what this actually does?

Reply:
The else behavior is for if their is no other window open at the time. If the user selects the window that is open the callback will trigger saying isExpanded is false. If no windows are open and the user expands one for the first time this callback will trigger and say that isExpanded is true.